analytics/analytics.go

```diff
@@ -6,18 +6,39 @@ import (
 	"strings"
 
 	"github.com/justsocialapps/holmes/assets"
+	"github.com/satori/go.uuid"
 )
 
+var analyticsRes []byte
+var etag string
+
+func init() {
+	etag = uuid.NewV4().String()
```

Comment:
Würde es nicht mehr Sinn machen, den etag als Hash in PrepareAnalytics aus dem byte-Array zu berechnen?

Reply:
Ohja, das würde es und ich habe diese Möglichkeit komplett übersehen. Dadurch könnte auch das UUID-Package wegfallen. Danke.